libsql-ffi/build.rs

```diff
@@ -124,127 +124,6 @@ pub fn build_bundled(out_dir: &str, out_path: &Path) {
         .flag("-D_POSIX_THREAD_SAFE_FUNCTIONS") // cross compile with MinGW
         .warnings(false);
 
-    if cfg!(feature = "wasmtime-bindings") {
-        cfg.flag("-DLIBSQL_ENABLE_WASM_RUNTIME=1");
-    }
-
-    if cfg!(feature = "bundled-sqlcipher") {
-        cfg.flag("-DSQLITE_HAS_CODEC").flag("-DSQLITE_TEMP_STORE=2");
-
-        let target = env::var("TARGET").unwrap();
-        let host = env::var("HOST").unwrap();
-
-        let is_windows = host.contains("windows") && target.contains("windows");
-        let is_apple = host.contains("apple") && target.contains("apple");
-
-        let lib_dir = env("OPENSSL_LIB_DIR").map(PathBuf::from);
-        let inc_dir = env("OPENSSL_INCLUDE_DIR").map(PathBuf::from);
```

Comment:
They will however require changes to compile for android as I was having errors on Cmake when trying to compile against sqlcipher.